src/main/resources/db/migration/V2__add_user_id_and_comment_owner_to_workspace_reason_comments.sql

```
@@ -0,0 +1,35 @@
+
+-- =============================================
+-- 1. workspace_reasons
+--    추가: workspace_request_id (FK → workspace_requests)
+--    NOTE: 기존 workspace_id, status 컬럼은 DB에 남겨둠 (Hibernate validate는 누락 컬럼만 검사)
+-- =============================================
+ALTER TABLE workspace_reasons
+    ADD COLUMN IF NOT EXISTS workspace_request_id BIGINT;
+
+ALTER TABLE workspace_reasons
+    ADD CONSTRAINT fk_workspace_reasons_request
+    FOREIGN KEY (workspace_request_id) REFERENCES workspace_requests(id);
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)